pkg/events/derive/hidden_kernel_module.go

```diff
@@ -0,0 +1,53 @@
+package derive
```

Comment:
Why does this event needs to be derived?
If all what you need is a place to process the arguments recieved from bpf code, a more proper place to do that will be the events processor.

Derived event mechanism should be used mostly for events we can't produce in bpf code since we need some extra data from userspace (e.g. parsing symbols or procfs)

Reply:
Need a place to only send the events for relevant hidden modules (by using the lru.Cache).
If I can "drop" the events without setting an error, then sure. Is it possible to do so via the events processor mechanism?

Reply:
It doesn't matter if you place the lru cache here or there.
The advantage of having this event as a non derived event will become clear when we will add actions in bpf code. Then it will be possible to block the operation in bpf code, and not wait for the event to be derived, where it is too late to block. Anyway, it is ok if you prefer to keep it here for now.